ABIDE_I/data_convert.py

```diff
@@ -0,0 +1,38 @@
+"""ABIDE I phenotype data conversion from CSV to TSV format.
+
+This script finds the data file as downloaded from http://www.nitrc.org/frs/downloadlink.php/4912 in .csv format and
+converts it to .tsv format file to be in compliance with BIDS specification. The TSV files are deposited in the
+`phenotype/` directory along with the data dictionary in JSON format.
+"""
+
+import argparse
+import csv
+from pathlib import Path
+
+import pandas as pd
+
+
+def help_text():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    # calling help prompt
+    help_text()
```

Comment:
I didn't see (and there shouldn't be) any arguments. What is this for?

Reply:
@ericearl This displays a brief description of what the script is doing even if it doesn't take arguments. Here's what it looks like:
```
% python ABIDE_I/data_convert.py -h
usage: data_convert.py [-h]

ABIDE I phenotype data conversion from CSV to TSV format.

This script finds the data file as downloaded from http://www.nitrc.org/frs/downloadlink.php/4912 in .csv format and
converts it to .tsv format file to be in compliance with BIDS specification. The TSV files are deposited in the
`phenotype/` directory along with the data dictionary in JSON format.

optional arguments:
  -h, --help  show this help message and exit
```

Reply:
I like this idea, but I fear this behavior may lead to people thinking they've run the script incorrectly with no arguments though that is the intent. I like the philosophy of succeed silently, fail loudly.

Reply:
@ericearl I don't feel strongly about this but as a python programmer I've gotten used to doing `python <program-name> -h` before I do anything else with the script. So I think it's nice to have some help text even there are no arguments. Also, there's little risk of it being interpreted as an error if they just read the text, no? In any case, I think based on our recent meeting we'd have an argparse function to take in input/output paths, so I suppose it doesn't matter now but I'm open to hearing any other thoughts you might have about programs with only `-h` argument.